### PR TITLE
Debot get info

### PR DESCRIPTION
**Pull request by Keshoid**
Support for 2 mandatory debot functions in dengine:
- `getRequiredInterfaces`
- `getDeBotInfo`
Details are here: https://github.com/tonlabs/DeBot-IS-consortium/blob/main/MandatoryDeBotFunctions.md

### CODE REVIEW

**Review by AlexeyVavilin — CHANGES_REQUESTED** [review]

File: ton_client/src/tests/contracts/abi_v2/helloDebot.sol [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        string name, string version, string publisher, string key, string author,
+        address support, string hello, string language, string dabi
+    ) {
+        name = "HelloWorld";
+        version = "0.2.0";
+        publisher = "TON Labs";
+        key = "Start develop DeBot from here";
+        author = "TON Labs";
+        support = address.makeAddrStd(0, 0x841288ed3b55d9cdafa806807f02a0ae0c169aa5edfe88a789a6482429756a94);
+        hello = "Hello, i'am a helloworld DeBot.";
```
[the remark this answers is not in the document]
Comment (Keshoid) [reply ton_client/src/tests/contracts/abi_v2/helloDebot.sol:30]: Fixed

File: ton_client/src/tests/contracts/abi_v2/helloDebot.sol [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        dabi = m_debotAbi.get();
+    }
+
+    function getRequiredInterfaces() public view override returns (uint256[] interfaces) {
+        return [ Terminal.ID ];
+    }
+
+    function setUserInput(string value) public {
+        // TODO: continue DeBot logic here...
+        Terminal.print(0, format("You entered \"{}\"", value));
```
[the remark this answers is not in the document]
Comment (Keshoid) [reply ton_client/src/tests/contracts/abi_v2/helloDebot.sol:41]: Fixed

File: ton_client/src/debot/mod.rs [hunk 3]
```diff
@@ -200,11 +269,18 @@ pub async fn execute(context: Arc<ClientContext>, params: ParamsOfExecute) -> Cl
         .map_err(Error::execute_failed)
 }
 
+/// [UNSTABLE](UNSTABLE.md)
+#[derive(Serialize, Deserialize, ApiType, Default)]
```
[the remark this answers is not in the document]
Comment (Keshoid) [reply ton_client/src/debot/mod.rs:273]: Fixex

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,15 @@ All notable changes to this project will be documented in this file.
 - [`utils.compress_zstd`](docs/mod_utils.md#compress_zstd) compresses data using Facebook's Zstandard algorithm.
 - [`utils.decompress_zstd`](docs/mod_utils.md#decompress_zstd) decompresses data using Facebook's Zstandard algorithm.
 
+### Changed
+- **Debot Module**:
+    - [breaking] `fetch` function does't create an instance of debot. It returns DeBot metadata (`DeBotInfo`).
```
Comment (AlexeyVavilin) [inline CHANGELOG.md:13]: Structure name is `DebotInfo`. Pay attention to names case

**Review by AlexeyVavilin — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (Keshoid) [reply CHANGELOG.md]: FIxed

**Review by AlexeyVavilin — APPROVED** [review]

**Pull request merged**